doc/lua_api.txt

```diff
@@ -1069,7 +1069,10 @@ The function of `param2` is determined by `paramtype2` in node definition.
     * Values range 0 - 23
     * facedir / 4 = axis direction:
       0 = y+,   1 = z+,   2 = z-,   3 = x+,   4 = x-,   5 = y-
-    * facedir modulo 4 = rotation around that axis
+    * The node is rotated 90 degrees around the X or Z axis so that its top face
+      points in the desired direction. For the y- direction, it's rotated 180
+      degrees around the Z axis.
+    * facedir modulo 4 = rotation around that axis; always a left-handed rotation.
```

Comment:
What would you think about the following? I believe it's a bit easier to understand, but will let the decision to others.

```
    * facedir / 4 = new direction of the original y+ axis:
      0 = y+,   1 = z+,   2 = z-,   3 = x+,   4 = x-,   5 = y-
    * facedir modulo 4 = left-handed rotation around the specified axis, in 90° steps.
```
This also would solve the special y- direction note that I honestly do not understand.

PS: Confirmed this using the param2 tool in the DevTest game.

Reply:
>This also would solve the special y- direction note that I honestly do not understand.

There are multiple (4) ways how one can rotate one direction to another one.

I.e. you could rotate +y to -y by rotating 180° around the x-axis, the difference would be that one rotation keeps the furnace front face in front (but rotated 180°) and the other one brings the back side to the front.

The details are important if you want to make anything with facedir (see [here](https://github.com/Desour/dslib/blob/71e4d9196d56204534826a10e5bbc97433c3a65d/src/rotnum.lua#L525-L529) if you want to see an example for a mod using this info).
The whole point of this PR is to document facedir, removing the need to read the src, right?

Reply:
I'd still keep the explanation of facedir modulo 4:
```
* facedir modulo 4 = left-handed rotation around the specified axis, in 90° steps.
```